### PR TITLE
data: Sduttt

### PR DESCRIPTION
**Pull request by Sduttt**

### CODE REVIEW

File: data/Sduttt.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            "name": "Twitter: Follow me",
+            "url": "https://twitter.com/Subhamd88404337",
+            "icon": "FaTwitter",
+            "color": "#00ACEE"
+        }
+    ],
+    "milestones": [
+        {
+            "title": "Joined GitHub",
+            "date": "March 20122",
```
Comment (weilincheng) [inline data/Sduttt.json:53]: The date value appears to be incorrect. Did you mean to use 2022 instead?

**Review by weilincheng — CHANGES_REQUESTED** [review]
Looks good to me.

File: data/Sduttt.json (hunk 1 above)
Comment (Sduttt) [reply data/Sduttt.json:53]: Yes!! Sorry for the typing mistake.

File: data/Sduttt.json [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    "links": [
+        {
+            "name": "My Portfolio",
+            "url": "https://sduttt.netlify.app/",
+            "icon": "FaLink"
+        },
+        {
+            "name": "Hashnode: Read my Articles",
+            "url": "https://sdutta.hashnode.dev/",
+            "icon": "FaBlog",
```
Comment (imbeshat) [inline data/Sduttt.json:34]:
```suggestion
            "icon": "SiHashnode",
```

File: data/Sduttt.json [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            "date": "March 20122",
+            "icon": "FaGithub",
+            "color": "grey",
+            "url": "https://github.com/Sduttt",
+            "description": "Contribute to OpenSource repos"
+        },
+        {
+            "title": "Started writing Technical Blogs",
+            "date": "July 2022",
+            "icon": "FaBlog",
```
Comment (imbeshat) [inline data/Sduttt.json:62]:
```suggestion
            "icon": "SiHashnode",
```

**Review by imbeshat — COMMENTED** [review]
For Hashnode, the icon of hashnode will be better.

**Review by bkpecho — COMMENTED** [review]
It's so fun to learn about the possibilities of LinkFree!

File: data/Sduttt.json (hunk 1 above)
Comment (weilincheng) [reply data/Sduttt.json:53]: Hi @Sduttt, do you want to push a commit to update the date? 😃

Comment (Sduttt) [reply data/Sduttt.json]: Hey @weilincheng , I have pushed the suggested changes.

File: data/Sduttt.json (hunk 3 above)
Comment (Sduttt) [reply data/Sduttt.json]: Thank you, Updated the changes.

**Review by weilincheng — APPROVED** [review]

File: data/Sduttt.json (hunk 1 above)
Comment (weilincheng) [reply data/Sduttt.json]: Awesome! I have approved the changes.

**Review by SurajPratap10 — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Great. Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**